geonode/settings.py

```diff
@@ -1078,13 +1081,7 @@
     DEFAULT_BACKEND_UPLOADER = {'geonode.importer'}
 """
 UPLOADER = {
-    "BACKEND": os.getenv("DEFAULT_BACKEND_UPLOADER", "geonode.importer"),
-    "OPTIONS": {
-        "TIME_ENABLED": ast.literal_eval(os.getenv("TIME_ENABLED", "False")),
```

Comment:
were option used in the old flow to configure geoserver and used in the upload template. Since we dont need them anymore was better to clean a bit the settings.py files.
Later with the new metadata editor, will be possible to remove also the old HTML files